packages/main/src/TimePickerPopoverTemplate.tsx

```diff
@@ -71,3 +76,31 @@ export default function TimePickerPopoverTemplate(this: TimePicker) {
 		</>
 	);
 }
+
+function valueStateMessage(this: TimePicker) {
+	return (
+		<>
+			{ this.shouldDisplayDefaultValueStateMessage ? this.valueStateDefaultText : <slot name="valueStateMessage">{this.valueStateMessage[0].innerHTML}</slot> }
```

Comment:
This is a hack - you must not clone the valueStateMessage - slot the user content directly